src/Actions/PurchaseInvoice/CreateOrderFromPurchaseInvoice.php

```diff
@@ -26,6 +31,21 @@ public static function models(): array
 
     public function performAction(): Order
     {
+        $layOutUserId = Arr::pull($this->data, 'lay_out_user_id');
+
+        if (
+            ! $layOutUserId
+            && data_get($this->data, 'iban')
+            && app(ContactBankConnection::class)->query()
+                ->where('contact_id', data_get($this->data, 'contact_id'))
```

Comment:
no need for data_get(), can just do $this->data['contact_id'], because its a required field and already validated at this point.